CHANGELOG_PENDING.md

```diff
@@ -24,6 +24,7 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 
 ### BUG FIXES:
 
+- [rpc] \#3935 Create buffered subscriptions on `/subscribe` (@melekes)
```

Comment:
nit, but do you want to make this a link like the other items in this list?

Reply:
technically this is the correct way according to the contributing.md and then there is a script to add all the links prior to release. People have been putting links recently, I guess we should decide on one way or the other?

Reply:
yeah, there's a script for that